README.md

```diff
@@ -11,7 +11,13 @@ For the MIPS official toolchain see
 	$ make CROSS_COMPILE=<toolchain-prefix> pistachio_marduk_defconfig
 	$ make CROSS_COMPILE=<toolchain-prefix>
 
-This will generate u-boot-pistachio-nor.img
+This will generate 'u-boot-pistachio_<board_name>-<tag_version>.img'
```

Comment:
This should be `u-boot-pistachio_<board_name>-<tag-version>.img`
it should be ` not, '